backend/src/API/mailAPI.ts

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios';
+import { Request } from 'express';
+import getEmailTransporter from '../nodemailer';
+import { isProd } from '../api';
+import AdminsDao from '../dao/AdminsDao';
+
+export const sendMail = async (to: string, subject: string, text: string): Promise<unknown> => {
+  // Don't send email notifications locally
+  if (!process.env.isProd) {
+    return {};
```

Comment:
Should there be some feedback still if its local to let you know its working?

Reply:
Good point, I'll add that.